site/docs/integrations/docker-extension.md

```diff
@@ -2,4 +2,121 @@
+
+3. **Getting shareable URLs**
+   
+   As soon as you log in, you will be able to see a list of running docker compose applications and all the services that are running in them. To get a public shareable URL for every service, turn on the toggle next to the compose application name. After that, you will be prompted to choose the access level. 
+
+   ![Livecycle Docker Extension](docker-ext-3.png)
+
+   You can choose between public and private access. If you choose public access, you will get a public URL that you can share with anyone. If you choose private access, you will get a private URL that require authentication and can only by used by your organization mebers. Then click on the "Share" button to get the shareable URL.
+
+   ![Livecycle Docker Extension](docker-ext-4.png)
+
+4. **Accessing the shared URL**
+
+   URLs created by the extension are consistent, shareable and can be use by a browser or any other http client.
+   Using these URLs, your team members will be able to see and interact with your local version of the app as long as the tunnel is open your workstation is running.  
+   Private environments require adding team members to your organization, and upon access, your team members will be prompt to authenticate.
+   
+   ![Livecycle Docker Extension](docker-ext-5.png)
+
+5. **Accessing Livecycle dashboard**
+
+   You can also access the Livecycle dashboard to see the logs and debug your application. Click on the "Open Link" button to open the Livecycle dashboard. On the dashboard, you can see all the running applications and services.
+   The Livecycle dashboard requires authentication and organization membership, similiarly to private environments/services 
+
+   ![Livecycle Docker Extension](docker-ext-6.png)
+
+6. **Debugging, inspecting, and logging**
+
+   Once you have opened the Livecycle dashboard, you can see all the environments/apps that are running. Click on the name of the environment for which you want to see the logs, terminal, etc. You can view the logs, terminal, and container inspection for each service.
+
+   ![Livecycle Docker Extension](docker-ext-7.png)
+
+That's it! You have successfully installed the Livecycle Docker Extension and shared your local development environment with your team.
+
+## FAQ
+
+<details>
+  <summary>What is the difference between the Livecycle Docker Extension and Ngrok?</summary>
+  
+  Livecycle Docker extesnion is integrated with Docker, and provides a smoother experience for Docker users.  
+  Consistent URLs, private environments, organizations and Google/Github authentication are supported out of the box.  
+  Livecycle dashboard provides debugging capabilites that includes log inspection, shell access, and container inspection.  
+  Lastly, the integration with Preevy allow you to create remote ephemeral environments that can be used when your workstation is offline or futher in your SDLC (preview environments for Pull Requests).  
```

Comment:
IMO further in your SDLC is too jargon

Reply:
I don't like it that much, as well, but it's the best I could come up with to indicate it can be used further in the development process.